moveit_core/robot_trajectory/test/test_robot_trajectory.cpp

```diff
@@ -538,7 +538,7 @@ TEST_F(RobotTrajectoryTestFixture, RobotTrajectorySmoothness)
 
   const auto smoothness = robot_trajectory::smoothness(*trajectory);
   ASSERT_TRUE(smoothness.has_value());
-  EXPECT_GT(smoothness.value(), 0.0);
+  EXPECT_EQ(smoothness.value(), 0.0);
```

Comment:
@sjahr is this change OK? It seems correct, to me, because `trajectory` contains 5 repeated waypoints, right?